README.md

```diff
@@ -428,14 +428,22 @@ Once installed, execute `py.test -vv` in the test directory to run all tests.
 We've added an installation file to make it easier to run tests and develop on
 Mac OSX. To install the dependencies on Mac OSX, run `install-mac-dev.sh`
 instead of `install.sh` and `install-dev.sh`.
-This will download Firefox ESR into the current folder, move geckodriver
+This will install Python packages in a local Python 3.6 virtualenv, 
+download Firefox Nightly into the current folder, move geckodriver
 next to the Firefox binary and install development dependencies.
 For the OpenWPM to be aware of which Firefox installation to run, set the
 FIREFOX_BINARY environment variable before running any commands.
 
-Example, running the OpenWPM tests on Mac OSX:
+Example, running a demo crawl on Mac OSX:
 
-    export FIREFOX_BINARY="$(PWD)/Firefox.app/Contents/MacOS/firefox-bin"
+    source venv/bin/activate
+    export FIREFOX_BINARY="$(PWD)/Nightly.app/Contents/MacOS/firefox-bin"
```

Comment:
Since we control the Firefox binary, it would be nice to automatically use this without having to `export FIREFOX_BINARY`. Could you file an issue for this? :)

Reply:
Sure :)

Reply:
Filed: https://github.com/mozilla/OpenWPM/issues/284